Gemfile.lock

```diff
@@ -182,64 +182,62 @@ GEM
       addressable (~> 2.7)
     letter_opener (1.7.0)
       launchy (~> 2.2)
-    listen (3.4.1)
+    listen (3.5.1)
       rb-fsevent (~> 0.10, >= 0.10.3)
       rb-inotify (~> 0.9, >= 0.9.10)
-    loofah (2.9.0)
+    loofah (2.9.1)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
     mail (2.7.1)
       mini_mime (>= 0.1.1)
-    marcel (0.3.3)
-      mimemagic (~> 0.3.2)
+    marcel (1.0.1)
     method_source (1.0.0)
-    mimemagic (0.3.5)
-    mini_mime (1.0.2)
-    minitest (5.14.3)
+    mini_mime (1.0.3)
+    minitest (5.14.4)
     msgpack (1.4.2)
     net-scp (3.0.0)
       net-ssh (>= 2.6.5, < 7.0.0)
     net-ssh (6.1.0)
-    nio4r (2.5.5)
-    nokogiri (1.11.1-x86_64-darwin)
+    nio4r (2.5.7)
+    nokogiri (1.11.3-x86_64-darwin)
       racc (~> 1.4)
     orm_adapter (0.5.0)
-    pagy (3.10.0)
+    pagy (3.11.0)
     parallel (1.20.1)
-    parser (3.0.0.0)
+    parser (3.0.1.0)
       ast (~> 2.4.1)
     pg (1.2.3)
-    popper_js (1.16.0)
+    popper_js (2.8.6)
     public_suffix (4.0.6)
-    puma (5.2.1)
+    puma (5.2.2)
```

Comment:
### [puma](https://puma.io)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 5.2.1 → 5.2.2

[_[change-log](https://github.com/puma/puma/blob/master/History.md), [source-code](https://github.com/puma/puma)_]





<details>
<summary>Commits</summary>

A change of **8** commits. See the full changes on [the compare page](https://github.com/puma/puma/compare/v5.2.1...v5.2.2).

These are the individual commits:
- (56512ba) [Update architecture.md](https://github.com/puma/puma/commit/56512ba3d7820445bc2ac11cfc646fe79811c562)
- (26776c8) [GitHub Actions now supports skipping tests](https://github.com/puma/puma/commit/26776c86dfdef24372b43ba3dab213754334a4f7)
- (7c91d90) [Add `#flush` and `#sync` methods to `Puma::NullIO` (#2553)](https://github.com/puma/puma/commit/7c91d90a1115080e940e0f748974acdefbe05ef8)
- (b0bd212) [Update check_changelog.yml](https://github.com/puma/puma/commit/b0bd2126079743a4e8caed6d098ca8dbc89e2f97)
- (1555ca2) [.rubocop.yml - exclude local bundles in test/worker_gem_inde…](https://github.com/puma/puma/commit/1555ca248eb46f9e2a008c922a004730799770a2)
- (7a2cdf6) [Restore sync=true on global stdout/stderr streams (#2557)](https://github.com/puma/puma/commit/7a2cdf6a7ea75c7271cece7268044d135d1025fb)
- (7970d14) [5.2.2 [ci skip]](https://github.com/puma/puma/commit/7970d14e63836d1c47a086928e533eee766af48d)
- (a192434) [Fix CI (#2561)](https://github.com/puma/puma/commit/a192434909513a0e77b6837ef208f627e502bed2)


</details>